man/execute_multi_array_udf.Rd

```diff
@@ -0,0 +1,37 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/execute.R
+\name{execute_multi_array_udf}
+\alias{execute_multi_array_udf}
+\title{TileDB Cloud UDF-Execution Helper for multiple-array UDFs}
+\usage{
+execute_multi_array_udf(
+  namespace,
+  array_list,
+  udf,
+  layout = NULL,
+  result_format = NULL
+)
+}
+\arguments{
+\item{namespace}{Namespace within TileDB cloud.}
+
+\item{array_list}{Names of the arrays. E.g. if one URI is \code{tiledb://hello/earth}
+and another is \code{tiledb://hello/mars} ##' then the namespace is "hello" and the array_list is
+\code{list("earth", "mars")}.}
+
+\item{udf}{An R function which takes an array of dataframes as argument.}
+
+\item{queryRanges}{List of two-column matrices, one matrix per dimension,
```

Comment:
queryRanges is not an argument above, and the arguments layout and result_format seem undefined.